### DISCUSSION TITLE
Country Code by Country name

### DISCUSSION THREAD

**Discussion by oc777** [FortnoxAB/csharp-api-sdk#241] [Q&A] [answered]
Is there any tested method of getting the `CountryCode` from `County` field?

For example, in `Order` class there is a field `DeliveryCountry`, but the `DeliveryCountryCode` is not passed through API.
An option could be to look up `Customer` and its `DeliveryCountryCode`, but there is no guarantee that all deliveries are going to be done within the specified country. 
Any ideas? 

The best, naturally, would be receiving this info directly from API, but for some reason the developers didn't think it was worth including in the `Order` obj :)

**Comment by richardrandak** [comment, marked as answer]
![Screenshot_1](https://user-images.githubusercontent.com/58220739/162392419-6d034731-4a1c-4a0c-9ce6-2f114ecea6de.png)

One way would be to make this kind of dictionary on your client. :) 
It would be easy to get English names with code, e.g. [here](https://gist.github.com/keeguon/2310008). Maybe you can find also an existing swedish list and use it.. or just define it manually for the subset of reasonable countries.

**Comment by oc777** [comment]
Yes, this could be a solution. It is somewhat error-prone, in case some country name would be updated/changed in Fortnox. (So I still think that the proper way would be to get CountryCode directly from API)

BTW, Have you noticed that there is no difference between Swedish and English names in the screenshot you attached? Both columns are in English. Same happens if I switch to Swedish language in GUI - both columns are in Swedish then :)

**Reply by richardrandak** [reply]
Naturally, it would be best for the API designers to include it, but that's something I can't help you with :)

When I switch to Swedish, it seems to be fine.. anyway, the English name is always chosen. However, some older customers I've created (probably by tests) has swedish country name, which makes me think it is arbitrary when set through the API.
